tests/core/editable/keystrokes/delbackspacequirks/manual/noselection.md

```diff
@@ -0,0 +1,15 @@
+@bender-tags: 4.7.0, tc, 13096
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea
+
+1. Open Chrome browser.
```

Comment:
As I stated in the comment, restricting this test to one browser is not the best option. The test should be changed to use more generic method.